src/pages/en/guides/cms/storyblok.md

```diff
@@ -203,13 +206,13 @@ const content = data.story.content;
 <StoryblokComponent blok={content} />
 ```
 
-To query our data, we will use the `useStoryblokApi` hook. This will initialize a new client instance using our integration configuration.
+To query your data, use the `useStoryblokApi` hook. This will initialize a new client instance using our integration configuration.
```

Comment:
```suggestion
To query your data, use the `useStoryblokApi` hook. This will initialize a new client instance using your integration configuration.
```